ambry-frontend/src/main/java/com/github/ambry/frontend/NamedBlobPutHandler.java

```diff
@@ -276,10 +283,17 @@ private boolean isRetriable(Throwable throwable) {
      * After TTL update finishes, call {@link SecurityService#postProcessRequest} to perform
      * request time security checks that rely on the request being fully parsed and any additional arguments set.
      * @param blobInfo the {@link BlobInfo} to use for security checks.
+     * @param blobId the {@link String} to use for blob id.
      * @return a {@link Callback} to be used with {@link Router#updateBlobTtl(String, String, long)}.
      */
-    private Callback<Void> routerTtlUpdateCallback(BlobInfo blobInfo) {
-      return buildCallback(frontendMetrics.updateBlobTtlRouterMetrics, blobId -> {
+    private Callback<Void> routerTtlUpdateCallback(BlobInfo blobInfo, String blobId) {
+      return buildCallback(frontendMetrics.updateBlobTtlRouterMetrics, convertedBlobId -> {
```

Comment:
where is convertedBlobId used in this callback ?

Reply:
Think this is a placeholder for the input parameter.
Look at the previous code, it was below. It use 'blobId' as the input parameter which is not used here as well. I just changed that 'blobId' to 'convertedBlobId' to avoid the name conflicts.
```
private Callback<Void> routerTtlUpdateCallback(BlobInfo blobInfo) {
      return buildCallback(frontendMetrics.updateBlobTtlRouterMetrics, blobId -> {
   securityService.processResponse(restRequest, restResponseChannel, blobInfo, securityProcessResponseCallback());
      }, uri, LOGGER, finalCallback);
    }
```